index.html

```diff
@@ -3186,21 +3186,23 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-The value of the <code>termsOfUse</code> <a>property</a> tells the
-<a>verifier</a> what actions it is required to perform (an <em>obligation</em>),
-not allowed to perform (a <em>prohibition</em>), or allowed to perform (a
-<em>permission</em>) if it is to accept the <a>verifiable credential</a> or
-<a>verifiable presentation</a>.
+Examples for the value of the <code>termsOfUse</code> <a>property</a> might be
+to tell the <a>verifier</a> 
+
+<ul><li>
+what procedures or policies were used in issuing the verifiable credential,
+for example, by providing a pointer to the location where these procedures 
+or policies can be found, or naming the standard that defines them;
+</li>
+<li>
+which superior authority gave the issuer authority to issue this particular verifiable credential.
+</ul></li>
         </p>
+<p>
+The <code>termsOfUse</code> <a>property</a> SHOULD be automatically processable by the verifier.
```

Comment:
Not accepted. I think we need to discuss this large addition in the WG before we accept this change

Reply:
I think these few lines of editorial advisory are necessary, unless we include a complete and explicit example of how `the value of the <code>termsOfUse</code> <a>property</a>` can otherwise `be automatically processable by the verifier`, in order to satisfy the `SHOULD` in your text.

That said, I cannot create a distinct PR adding these few lines, nor a distinct issue referring to this text in the main document, until this PR is merged.